eng/pipelines/templates/jobs/tests.yml

```diff
@@ -0,0 +1,84 @@
+parameters:
```

Comment:
I know I suggested this name but I wonder if this will cause confusion because it doesn't map to the tests.yml file in the rest of the repo like the ci.yml maps. Maybe naming this file ci.tests.yml and the other one tests.yml might better map.

Reply:
How about ci.tests.yml and live.tests.yml?

Reply:
I could live with that as well.